### DISCUSSION TITLE
Github Copilot Student Free Access Issue

### DISCUSSION THREAD

**Discussion by Jiaqing-ASU** [community/community#50862] [GitHub Education] [answered]
Hello Everyone,

I am a user of both GitHub Pro and GitHub Copilot. I have paid those services by myself for several months. My friend just told me that those are free for students. Then I verified my student status. After this, I found there is a coupon for 2 years for my GitHub Pro which makes me use the GitHub Pro for free. But there is no discount coupon for my GitHub Copilot and there is nothing changed for my next GitHub Copilot billing. What should I do to use the GitHub Copilot for free as well? Do I need to cancel my current subscription and resubscribe the Copilot after my current service ends?

Any help is highly appreciated!!

**Comment by samus-aran** [comment, marked as answer]
Hi @Jiaqing-ASU 

The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). We suggest you reach out regarding to Account & Billing. Please open a ticket with them :)

**Reply by Jiaqing-ASU** [reply]
Thank you so much. I appreciate it. This has been solved.
